.gitignore

```diff
@@ -5,3 +5,4 @@ dist/
 livy.egg-info/
 .pytest_cache/
 .tox/
+.idea/
```

Comment:
I generally prefer not to include editor-specific ignores. Can you please remove this and add it instead to [a global ignores file](https://acroz.dev/2019/07/14/global-git-ignore/)?